CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+# 1.7.0-beta01 (August 2024)
+
+_Changes since release/1.7.0-alpha03_
+
+## Breaking changes
+### iOS
+- [`UIKitView` and `UIKitViewController` in `package androidx.compose.ui.interop` are deprecated. New API are mentioned in deprecation message. Deprecated invocations should work fine unless custom `onResize` is used, it is disallowed now and will print a warning.](https://github.com/JetBrains/compose-multiplatform-core/pull/1494)
+- [Actual of expected `InteropView` on iOS is `UIResponder` now instead of `UIView`. It's the first common ancestor for `UIViewController` and `UIView`, both of which can be integrated using iOS interop APIs](https://github.com/JetBrains/compose-multiplatform-core/pull/1489)
+- [The app will crash by default, if `CADisableMinimumFrameDurationOnPhone` is not set to true in `Info.plist`. Use newly added `ComposeUIViewControllerConfiguration.enforceStrictPlistSanityCheck` to opt-out of this behavior](https://github.com/JetBrains/compose-multiplatform-core/pull/1451)
+
+## Features
+### Multiple Platforms
+- [New multiplatform module "material-navigation"](https://github.com/JetBrains/compose-multiplatform-core/pull/1504)
+
+### iOS
+- [New `UIKitView` and `UIKitViewController` API in `package androidx.compose.ui.viewinterop`. Support of `onReset` to reuse the interop composable emitted node and avoid excessive native views reallocations, fine-grain touches strategy control (cooperative with explicit time delay, non-cooperative where no touches are received by Compose, ignoring touches)](https://github.com/JetBrains/compose-multiplatform-core/pull/1494)
+
+### Resources
+- [Added support of test resources in Compose Multiplatform projects](https://github.com/JetBrains/compose-multiplatform/pull/5122)
+- [Added support of multi-module resources in JVM-only projects](https://github.com/JetBrains/compose-multiplatform/pull/5122)
+
+## Fixes
+### Multiple Platforms
+- [_(prerelease fix)_ Fix redirect on android artifacts for "window-core" module](https://github.com/JetBrains/compose-multiplatform-core/pull/1506)
+
+### Desktop
+- [_(prerelease fix)_ Fixed drag-and-drop not working after a popup is displayed in the window](https://github.com/JetBrains/compose-multiplatform-core/pull/1493)
+
+### Resources
+- [_(prerelease fix)_ Fix a resource reading on iOS 12](https://github.com/JetBrains/compose-multiplatform/pull/5123)
+- [_(prerelease fix)_ Fix resource reading on Java 11](https://github.com/JetBrains/compose-multiplatform/pull/5125)
+
+### Gradle Plugin
+- [Internal refactor to support project isolation](https://github.com/JetBrains/compose-multiplatform/pull/5120)
```

Comment:
```suggestion
- [Support project isolation](https://github.com/JetBrains/compose-multiplatform/pull/5120)
```

Reply:
should it be excluded? I looked more thoroughly after your comment, this PR changes only internal code

Reply:
No, it brings support for the upcoming Gradle feature. But release notes should be only about public things.
Support project isolation - is public affecting and should be mentioned
Internal refactor - is clearly internal and should NOT be mentioned

In case if some refactor fixes some issue - it should be like "Fixes X use case"